<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="1" file="validate_colors_list_tests.py" name="validate_colors_list_tests.TestValidateColorsList-20191013152317" skipped="0" tests="7" time="0.011" timestamp="2019-10-13T15:23:17">
	<testcase classname="validate_colors_list_tests.TestValidateColorsList" file="src/unittest/python/validate_colors_list_tests.py" line="19" name="test3ColorBands" time="0.010" timestamp="2019-10-13T15:23:17">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="validate_colors_list_tests.TestValidateColorsList" file="src/unittest/python/validate_colors_list_tests.py" line="9" name="test4ColorsBands" time="0.000" timestamp="2019-10-13T15:23:17">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="validate_colors_list_tests.TestValidateColorsList" file="src/unittest/python/validate_colors_list_tests.py" line="13" name="test5ColorBands" time="0.000" timestamp="2019-10-13T15:23:17">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="validate_colors_list_tests.TestValidateColorsList" file="src/unittest/python/validate_colors_list_tests.py" line="33" name="test5ValidColors" time="0.000" timestamp="2019-10-13T15:23:17">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="validate_colors_list_tests.TestValidateColorsList" file="src/unittest/python/validate_colors_list_tests.py" line="39" name="testInvalidColorLimegreen" time="0.000" timestamp="2019-10-13T15:23:17">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="validate_colors_list_tests.TestValidateColorsList" file="src/unittest/python/validate_colors_list_tests.py" line="46" name="testInvalidToleranceBandBlack" time="0.000" timestamp="2019-10-13T15:23:17">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="validate_colors_list_tests.TestValidateColorsList" file="src/unittest/python/validate_colors_list_tests.py" line="24" name="test6ColorBands" time="0.000" timestamp="2019-10-13T15:23:17">
		<failure message="ValueError not raised" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/home/student/Desktop/SystemProgramming/Projects/NolanWilliamsProject2/src/unittest/python/validate_colors_list_tests.py", line 27, in test6ColorBands
    validateColorsList(listOfColors)
AssertionError: ValueError not raised
]]>		</failure>
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
</testsuite>
